sources/react-app/src/shared/DynamicRoute.js

```diff
@@ -29,15 +29,25 @@ const limit = 3;
 export default function DynamicRoute(props) {
   const { match, location } = props;
   const [state, setState] = useState(null);
-  const url = match.path.includes(':')
+  let url = match.path.includes(':')
     ? match.path.substring(0, match.path.indexOf(':') -1)
     : match.url;
 
+  if (url.includes('/post')) {
+    url += '.xml';
+  }
+
   useEffect(() => {
     let destroyed = false;
     let page = parseInt(parse(location.search).page ?? 1) - 1;
     const pagination = { limit, offset: page ? (page * limit) : 0 };
-    reportNavigation(url);
+
+    if (window.crafterRequire) {
+      window.crafterRequire(['guest'], function (guest) {
+        guest.reportNavigation(location, location.pathname);
+      });
+    }
```

Comment:
use/test reportNavigation from sdk

Reply:
changed to reportNavigation, there seems to be an issue in the studio side, will check/report